### PR TITLE
None description shouldnt be updated

### PR DESCRIPTION
**Pull request by korgan00**

### CODE REVIEW

**Comment by Tansito** [comment]
I think you need to run `tox -eblack` to fix the formatting, @korgan00

File: gateway/api/serializers.py [hunk 1]
```diff
@@ -117,7 +117,11 @@ def update(self, instance, validated_data):
         instance.artifact = validated_data.get("artifact")
         instance.author = validated_data.get("author")
         instance.image = validated_data.get("image")
-        instance.description = validated_data.get("description")
+
+        description = validated_data.get("description")
+        if description is not None:
```
Comment (akihikokuroda) [inline gateway/api/serializers.py:122]: Nit: in Python
`if description:` is same as `if description is not None:` in most of the cases and it's simpler

**Review by akihikokuroda — APPROVED** [review]
There is one nit comment.  Other than that, LGTM!

**Review by Tansito — APPROVED** [review]
I agree with @akihikokuroda comment, apart from that LGTM too 👍

File: gateway/api/serializers.py (hunk 1 above)
Comment (Tansito) [reply gateway/api/serializers.py:122]: thanks for the fast test, @korgan00 , understood!

**Pull request merged**